src/SwedbankPay/Api/Client/Client.php

```diff
@@ -343,7 +343,7 @@ protected function processRequestResult()
             "Response Body:\n%s\n\n" .
             "<<<<<<<< END PAYEX API CLIENT REQUEST DEBUG INFO <<<<<<<<\n\n",
             $this->getMethod(),
-            $this->getBaseUrl() . $this->getEndpoint(),
+            str_replace('%s', '*', $this->getBaseUrl() . $this->getEndpoint()),
```

Comment:
Is this `str_replace` necessary?

Reply:
@asbjornu There was a bug where the script had failure if someone will define url with `%s`